gallery/src/components/NxDropdown/NxDropdownPage.tsx

```diff
@@ -260,15 +260,35 @@ const NxDropdownPage = () =>
                 any <NxCode>.nx-dropdown-right-button</NxCode>s.
               </NxTable.Cell>
             </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+        <NxWarningAlert>
+          The <NxCode>nx-dropdown-icon</NxCode> class name has been deprecated.
```

Comment:
Since we're considering this PR to be a breaking change, I don't think we need to mention `nx-dropdown-icon` anymore at all.  Especially since icons which previously used that class no longer have to use any class at all to achieve the same effect (though they are still safe to use it, it just doesn't do anything anymore)

Reply:
> Since we're considering this PR to be a breaking change, I don't think we need to mention `nx-dropdown-icon` anymore at all. Especially since icons which previously used that class no longer have to use any class at all to achieve the same effect (though they are still safe to use it, it just doesn't do anything anymore)

Removed the deprecated warning: 723a56df94ba202e58d35fe44c909e0c581f8657